python/dgl/function/message.py

```diff
@@ -235,15 +235,15 @@ def src_mul_edge(src, edge, out):
     ----------
     src : str
         The source feature field.
-    dst : str
-        The destination feature field.
+    edge : str
+        The edge feature field.
     out : str
         The output message field.
 
     Examples
     --------
     >>> import dgl
-    >>> message_func = dgl.function.src_mul_edge('h', 'h', 'm')
+    >>> message_func = dgl.function.src_mul_edge('h', 'e', 'm')
```

Comment:
"src_mul_edge" -> "u_mul_e"

Reply:
wait... shouldn't we at least keep the docstring consistent with the function, even it is deprecated?

Reply:
I didn't realize that:)